src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSMarshalerType.cs

```diff
@@ -37,6 +38,7 @@ private JSMarshalerType(JSFunctionBinding.JSBindingType signatureType)
         /// <summary>
         /// Marshaling ignores result of the JavaScript function.
         /// </summary>
+        /// <returns>Marshaler metadata</returns>
```

Comment:
Are all these APIs returning exactly the same thing? Could you instead elaborate on what they represent?

Reply:
The structure of the object is intentionally undocumented because we don't want users to try take dependency on it's internal layout.